src/models/User.ts

```
@@ -52,15 +52,14 @@ const PROFILE_FIELDS = [
   "locale",
 ];
 
-async function getProfile(
-  storage: DurableObjectStorage,
-): Promise<Profile | null> {
-  const profilesString = await storage.get<string>(StorageKeys.profile);
-  if (!profilesString) {
+async function getProfile(storage: DurableObjectStorage) {
+  const jsonData = await storage.get<string>(StorageKeys.profile);
+
+  try {
+    return ProfileSchema.parse(JSON.parse(jsonData as string));
```

that is cool!

Do we want to just swallow the error though? What would an error mean here?